anvil/src/drawing.rs

```diff
@@ -103,7 +104,12 @@ where
             CursorImageStatus::Surface(surface) => {
                 let elements: Vec<PointerRenderElement<R>> =
                     smithay::backend::renderer::element::surface::render_elements_from_surface_tree(
-                        renderer, surface, location, scale, alpha,
+                        renderer,
+                        surface,
+                        location,
+                        scale,
+                        alpha,
+                        Some(Kind::Cursor),
```

Comment:
I wonder if maybe it's a good idea to set kind separately so it's not spammed into every invocation of these functions. Since it's something you set on individual elements generally.

Reply:
Currently no render element has a public setter, everything is done a construction. Setting it after construction would require a mutable reference, not sure if this would make a good API. It would work to create a "overload" function, something like `render_element_from_surface_tree_with_kind`, but tbh I am not sure if this would be an improvement.